HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -95,10 +95,13 @@
   </sch:pattern>
 
   <sch:pattern>
-    <sch:title>[Address]</sch:title>
-    <sch:rule context='/h:HPXML/h:Building/h:Site/h:Address'>
-      <sch:assert role='ERROR' test='count(h:StateCode) &lt;= 1'>Expected 0 or 1 element(s) for xpath: StateCode</sch:assert>
-      <sch:assert role='ERROR' test='count(h:ZipCode) &lt;= 1'>Expected 0 or 1 element(s) for xpath: ZipCode</sch:assert>
+    <sch:title>[BuildingSite]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:Site'>
```

Comment:
If the Site element exists, HPXML requires a SiteID element. Can you add that? I think it would be:
`<sch:assert role='ERROR' test='count(h:SiteID) = 1'>Expected 1 element(s) for xpath: SiteID</sch:assert>`

Reply:
What about BuildingID?

Reply:
Should we add a `hpxml.header.site_id` like `hpxml.header.building_id`?

Reply:
Oh, I forgot that I somewhat recently went through and [removed all these HPXML-schema required IDs from EPvalidator.xml](https://github.com/NREL/OpenStudio-HPXML/commit/eaf9185114c4c49584d095b909a4fb49257de8ab#). You can ignore this request.

Reply:
What about the second question: allow BuildResidentialHPXML to set the SiteID (similar to BuildingID)?

Reply:
Nah, no need to.